packages/openapi-ts/src/compiler/index.ts

```diff
@@ -31,6 +32,11 @@ export class TypeScriptFile {
         this._imports = [...this._imports, compiler.import.named(...params)];
     }
 
+    public setPath(path: string) {
```

Comment:
Can we pass the path in the class contructor (same with a header)? Its probably cleaner than doing this. And they can be optional